### PR TITLE
New Documentation

### PR DESCRIPTION
**Pull request by cczhu**
Updated documentation to new structure.  Moved __init__ docstrings to their respective file format files in the docs.  Created tutorial on using metaclassed VDIF header to generate new EDVs, and corrupted header VDIF file to support tutorial.

There are two DRAOVDIFHeader class definitions (one that overrides `fromfile`, the other appending additional commands to `__init__`), but I feel like the second version is superior to the first, and simpler to explain.  Maybe remove the first?

### CODE REVIEW

**Comment by mhvk** [comment]
@cczhu - I liked the new documentation a lot, but also had quite a number of mostly small comments. I implemented those directly. I also made more major edits to the `new_edv` tutorial (and put it in a tutorials directory). Please have a look what you think. Overall, I think this is ready to go in...

File: docs/baseband/dada/index.rst [hunk 1]
```diff
@@ -20,9 +15,7 @@ Usage
 All files should be opened using :func:`~baseband.dada.open`.  Single files can
 be opened in binary mode, which provides a normal file reader but extended with
 methods to read a :class:`~baseband.dada.DADAFrame`.  For dada files, which
-consist of just a single header and payload, such frames contain all the data.
-
-::
+consist of just a single header and payload, such frames contain all the data.::
```
Comment (cczhu) [inline docs/baseband/dada/index.rst]: Sphinx builds this as " ...such frames contain all the data.: ", with a trailing colon, which is why I added the indented block flag after a newline.

Comment (mhvk) [reply docs/baseband/dada/index.rst]: In that case, I think just omit the period - a colon is fine.

**Comment by cczhu** [comment]
Agreed for all your changes (the new validity mask is much slicker!), plugged a few formatting and syntax holes.  We're good for merging the pull request, which I can do on my end.  Philosophically, are you for squashing or merging, considering some of the commits are minuscule?

**Comment by mhvk** [comment]
OK, let's merge!  Overall, I personally don't really mind multiple commits, though I like them to have some logic, and not include reversions. Certainly, this is all fine.

**Pull request merged**